pkgs/by-name/hy/hyprland-workspaces-tui/package.nix

```
@@ -0,0 +1,41 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+  makeWrapper,
+  hyprland-workspaces,
+}:
+rustPlatform.buildRustPackage rec {
+  pname = "hyprland-workspaces-tui-${version}";
+  version = "1.0.2";
+  src = fetchFromGitHub {
+    owner = "Levizor";
+    repo = "hyprland-workspaces-tui";
+    rev = "9eff75a4431193d1ebeb2e23bdefcd2dbcaa7666";
+    sha256 = "sha256-3QmqoyWmtC4ps8dtIWEoLjzdzKAXOujyz+GgOlo172Q=";
+  };
+
+  cargoHash = "sha256-RatW+9CPOmaG9u3/XCNnqyThBafZa0qRm02OvATq45I=";
+
+  nativeBuildInputs = [
+    makeWrapper
+  ];
+
+  buildInputs = [
+    hyprland-workspaces
+  ];
+
+  postFixup = ''
+    wrapProgram $out/bin/hyprland-workspaces-tui \
+      --set PATH ${hyprland-workspaces}/bin:$PATH
+  '';
+
+  meta = with lib; {
```

```suggestion
  meta = {
```
https://github.com/NixOS/nixpkgs/issues/292468#issuecomment-2016454242